code/modules/vending/vending.dm

```diff
@@ -1596,7 +1596,9 @@
 	"Prepared in moments!",
 	"I'm a chef who works 24 hours a day!")
 	var/sharpen = FALSE
-
+	create_products()
+		..()
+		product_list += new/datum/data/vending_product(/obj/item/reagent_containers/food/snacks/pizza/random, 100)
```

Comment:
While [this bit of code](https://github.com/goonstation/goonstation/blob/master/code/modules/vending/vending.dm#L786) is as-is, I wouldn't want this number over 10.

Reply:
I'm not sure what you mean?

Reply:
The way "malfunction" works is it seems to eject *all* of a given type of item. I may be misreading, but if that is correct then this would eject 100 pizzas at once.

Fine with changing to set to 0.

Reply:
Oh yeah you're right I didn't understand that line, but now that I understand it, I do want to set it to 10 so that explosions will eject some pizzas!
Testing `freestuff` changes right now

Reply:
We use ten pizzas so that when we get bombed there's some pizza (yay features!) and `freestuff` so that we can throw more than ten